packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -313,6 +327,39 @@ class _BottomSheetState extends State<BottomSheet> {
   }
 }
 
+class _DragHandle extends StatelessWidget {
+  const _DragHandle({required this.onSemanticsTap});
+  final VoidCallback onSemanticsTap;
+
+  @override
+  Widget build(BuildContext context) {
+
+    final BottomSheetThemeData bottomSheetTheme = Theme.of(context).bottomSheetTheme;
+    final BottomSheetThemeData m3Defaults =  _BottomSheetDefaultsM3(context) ;
+
+    final Size dragHandleSize= bottomSheetTheme.dragHandleSize ?? m3Defaults.dragHandleSize!;
+    final Color dragHandleColor= bottomSheetTheme.dragHandleColor ?? m3Defaults.dragHandleColor!;
+
+    return Semantics(
+      label: MaterialLocalizations.of(context).modalBarrierDismissLabel,
+      container: true,
+      onTap: onSemanticsTap,
+      child: Padding(
+        // Add padding to make sure target size is larger than 48dp.
+        padding: const EdgeInsets.fromLTRB(20, 15, 20, 30),
```

Comment:
(In my case, the advantage I found was that `kMinInteractiveDimension` was a convenient constant to refer back to elsewhere in my code, where I needed to express the space belonging to the drag handle. In particular, I wanted the scrollable content of the sheet (in a `DraggableScrollableSheet`) to start out below the drag handle in the y-direction, but then be scrollable under the drag handle in the z-direction, with help from a `Stack`. So then the contents get clipped when they scroll off the top of the sheet, instead of getting clipped by an artificial-looking line below the drag handle.)

Reply:
Thanks for the suggestion!  I do agree using kMinInteractiveDimension here is better than hardcode 22

Reply:
Great! 🙂 I see in the current revision 5f47c77fd a variable `_kMinInteractiveSize` has been added; is there a reason for using that instead of the existing global [`kMinInteractiveDimension`](https://api.flutter.dev/flutter/material/kMinInteractiveDimension-constant.html)?

Reply:
Hi Chris, I didn't realize that there is a existing global kMinInteractiveDimension, thanks for pointing it out to me! I will use it.